onnx_tf/doc/IMPLEMENTING_NEW_OP.md

```diff
@@ -0,0 +1,43 @@
+How to implement new op
+======
+
+When you get `{} op is not implemented`, you can follow next steps to implement.  
+Customize op can also be implemented in similar way.
+
+### Backend
+
+1.  Find specification from [onnx/Operators](https://github.com/onnx/onnx/blob/master/docs/Operators.md).
+2.  Decide if need a specific handler.  
+    Op doesn't need a specific handler when tf's specification highly matches onnx's, means:
+    
+    - inputs are same
+    - tf's attributes is a subset of onnx's    
+    - attr doesn't exist in tf could be set by `DEFAULT_ONNX_ATTR_PER_OP`  
```

Comment:
Might want to tell users that things like DEFAULT_ONNX_ATTR_PER_OP, ONNX_OP_TO_TF_OP are in common.py.